docs/core/deploying/trimming-options.md

```diff
@@ -9,27 +9,47 @@ ms.date: 08/25/2020
 
 The following MSBuild properties and items influence the behavior of [trimmed self-contained deployments](trim-self-contained.md). Some of the options mention `ILLink`, which is the name of the underlying tool that implements trimming. More information about the underlying tool can be found at the [Linker documentation](https://github.com/mono/linker/tree/master/docs).
 
+Trimming with `PublishTrimmed` was introduced in .NET Core 3.0. The other options are available only in .NET 5 and above.
+
 ## Enable trimming
 
 - `<PublishTrimmed>true</PublishTrimmed>`
 
    Enable trimming during publish, with the default settings defined by the SDK.
 
-When using `Microsoft.NET.Sdk`, this will perform assembly-level trimming of the framework assemblies from the netcoreapp runtime pack. Application code and non-framework libraries are not trimmed. Other SDKs may define different defaults.
+When using `Microsoft.NET.Sdk`, this will trim the framework assemblies from the netcoreapp runtime pack. Application code and non-framework libraries are not trimmed. Other SDKs may define different defaults.
```

Comment:
Not just `framework assemblies`, right? Any assembly marked with `IsTrimmable`, which all framework assemblies are annotated with.

Reply:
Clarified this